external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -136,6 +137,20 @@ class KafkaTestUtils(
     kdcConf.setProperty(MiniKdc.DEBUG, "true")
     kdc = new MiniKdc(kdcConf, kdcDir)
     kdc.start()
+    val krb5Conf = Source.fromFile(kdc.getKrb5conf, "UTF-8").getLines()
+    val rewriteKrb5Conf = krb5Conf.map(s => if (s.contains("libdefaults")) {
+      s + "\n" +
+        "    default_tkt_enctypes=aes128-cts-hmac-sha1-96\n" +
+        "    default_tgs_enctypes=aes128-cts-hmac-sha1-96 "
+    } else {
+      s
+    })
+    kdc.getKrb5conf.delete()
+    val writer = new PrintWriter(kdc.getKrb5conf)
+    // scalastyle:off
+    rewriteKrb5Conf.foreach(writer.println)
+    writer.flush()
```

Comment:
`java.io.Writer` calls `flush` in `close`.

Reply:
> `java.io.Writer` calls `flush` in `close`.

`flush()` removed.